apps/console/src/public/deployment.config.json

```diff
@@ -891,6 +891,7 @@
                 "scopes": {
```

Comment:
Check if all the RBAC related configs are added.
- J2 Mappings
- `default.json` updates
- `console` ui scopes registered